src/caf/distribute/gravity_model/multi_area.py

```diff
@@ -338,87 +499,29 @@ def _calibrate(
-            into the cost function as initial parameters.
-
-        running_log_path:
-            Path to output the running log to. This log will detail the
-            performance of the run and is written in .csv format.
-
-        target_cost_distribution:
-            The cost distribution to calibrate towards during the calibration
-            process.
-
-        diff_step:
-            Copied from scipy.optimize.least_squares documentation, where it
-            is passed to:
-            Determines the relative step size for the finite difference
-            approximation of the Jacobian. The actual step is computed as
-            `x * diff_step`. If None (default), then diff_step is taken to be a
-            conventional “optimal” power of machine epsilon for the finite
-            difference scheme used
-
-        ftol:
-            The tolerance to pass to `scipy.optimize.least_squares`. The search
-            will stop once this tolerance has been met. This is the
-            tolerance for termination by the change of the cost function
-
-        xtol:
-            The tolerance to pass to `scipy.optimize.least_squares`. The search
-            will stop once this tolerance has been met. This is the
-            tolerance for termination by the change of the independent
-            variables.
-
-        grav_max_iters:
-            The maximum number of calibration iterations to complete before
-            termination if the ftol has not been met.
-
-        failure_tol:
-            If, after initial calibration using `init_params`, the achieved
-            convergence is less than this value, calibration will be run again with
-            the default parameters from `self.cost_function`.
-
-        default_retry:
-            If, after running with `init_params`, the achieved convergence
-            is less than `failure_tol`, calibration will be run again with the
-            default parameters of `self.cost_function`.
-            This argument is ignored if the default parameters are given
-            as `init_params.
-
-        n_random_tries:
-            If, after running with default parameters of `self.cost_function`,
-            the achieved convergence is less than `failure_tol`, calibration will
-            be run again using random values for the cost parameters this
-            number of times.
-
-        verbose:
-            Copied from scipy.optimize.least_squares documentation, where it
-            is passed to:
-            Level of algorithm’s verbosity:
-            - 0 (default) : work silently.
-            - 1 : display a termination report.
-            - 2 : display progress during iterations (not supported by ‘lm’ method).
-
-        kwargs:
-            Additional arguments passed to self.gravity_furness.
-            Empty by default. The calling signature is:
-            `self.gravity_furness(seed_matrix, **kwargs)`
-
+        distributions: MultiCostDistribution,
```

Comment:
Docstring should explain all the parameters here because there is nowhere else for the user to see that information, I think the parameters will be simpler if the GMParams dataclass mentioned above is created because the details of each of those parameters would be documented there instead of here.